Dockerfile

```diff
@@ -2,11 +2,6 @@ ARG PARENT_VERSION=latest
 FROM p4lang/pi:${PARENT_VERSION}
 LABEL maintainer="Antonin Bas <antonin@barefootnetworks.com>"
 
-# Default to using 2 make jobs, which is a good default for CI. If you're
-# building locally or you know there are more cores available, you may want to
-# override this.
-ARG MAKEFLAGS=-j2
```

Comment:
do you think you could update https://github.com/p4lang/behavioral-model/blob/main/Dockerfile.noPI as well?

Reply:
done